### PR TITLE
[INT2-115] Add proposal prosess 

### PR DESCRIPTION
**Pull request by CRIPU**

### CODE REVIEW

File: governance/proposal-process.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+- [YIP 55: YIP導入・投票プロセスの正式化](https://gov.yearn.finance/t/yip-55-formalize-the-yip-process/7959)
+
+# 投票
+
+## 投票の方法
+
+- 以下の場所でYFIを保有またはステーキングすることで、Yearnの[スナップショット](https://snapshot.page/#/yearn)ページへの投票が可能になります。:
+	- 自分のウォレット
+	- YFI yVault V2 (yvYFIトークンを持っているのと同じ状態)
+	- Balancer YFI/WETH LP token
```
Comment (DarkGhost7) [inline governance/proposal-process.md:54]:
```
	- Balancer YFI/WETH LP token
```
maybe translate token?

File: governance/proposal-process.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+	- Uniswap YFI/WETH LP token
+	- Sushiswap YFI/WETH LP token staked in MasterChef
```
Comment (DarkGhost7) [inline governance/proposal-process.md:56]:
```
token staked in
```
Maybe translate?

File: governance/proposal-process.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+	- MakerDAO YFI collateral
+	- Unit Protocol YFI collateral
```
Comment (DarkGhost7) [inline governance/proposal-process.md:58]: ```collateral```
maybe translate

**Review by DarkGhost7 — CHANGES_REQUESTED** [review]
Translate more, but looks good other than that!

File: governance/proposal-process.md (hunk 1 above)
Comment (CRIPU) [reply governance/proposal-process.md]: BalancerのYFI/WETH LPトークン
I fixed it thank you

File: governance/proposal-process.md (hunk 2 above)
Comment (CRIPU) [reply governance/proposal-process.md]: I fixed it sorry > <

File: governance/proposal-process.md (hunk 3 above)
Comment (CRIPU) [reply governance/proposal-process.md]: 担保、I fixed it thanks!

**Pull request closed without merging**